test/unit/spec/EnrollChoices_spec.js

```diff
@@ -327,7 +336,7 @@ function (_, $, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
         itp('has a setup button for each unenrolled optional factor which navigates to the correct page (On-Prem)',
           function () {
           return setupWithAllOptionalSomeEnrolled(true).then(function (test) {
-            expect(test.form.factorButton('OKTA_VERIFY').length).toBe(1);
+            expect(test.form.factorButton('OKTA_VERIFY_PUSH').length).toBe(0);
```

Comment:
setupWithAllOptionalSomeEnrolled sets the first factor in the xhr to enrolled. In `MFA_ENROLLED_allFactors_OnPrem.js`, the first factor was changed from `QUESTION` to `OKTA_VERIFY_PUSH`. Because `OKTA_VERIFY_PUSH` is now enrolled, it will not have a factor button, so length will be 0. I decided to write it this way, so the test would ensure the first factor does not have a button, and the rest do. Previously the test only ensured the un-enrolled factors had a button.